ext/puma_http11/org/jruby/puma/Http11.java

```diff
@@ -127,7 +127,11 @@ public static void http_field(Ruby runtime, RubyHash req, ByteList buffer, int f
             }
         }
 
-        while (vlen > 0 && Character.isWhitespace(buffer.get(value + vlen - 1))) vlen--;
+        while (vlen > 0 && (buffer.get(value + vlen - 1) == ' ' || buffer.get(value + vlen - 1) == '\t')) vlen--;
```

Comment:
Same comment as the C code, we should use a builtin instead of hand rolling manually checking space and tab.

Reply:
See opening comment, paragraph right above 'Your checklist for this pull request'.  OWS space is defined as two characters, HTAB and SPACE.